lua/nvim-treesitter/parsers.lua

```diff
@@ -617,6 +617,16 @@ list.fsh = {
   maintainers = { "@mgramigna" },
 }
 
+list.fsharp = {
+  install_info = {
+    url = "https://github.com/ionide/tree-sitter-fsharp",
+    files = { "src/parser.c", "src/scanner.c" },
+    branch = "main",
+    location = "fsharp",
```

Comment:
That is a very weird layout.

Reply:
The repo also contains a parser for fsharp signature files. I just copied the layout of the typescript parser here tbh.

Reply:
Is the signature parser relevant here (as injections)?

Reply:
Eventually I would like it to be. But right now the signature file parser is very experimental.